fronts-client/src/types/Collection.ts

```diff
@@ -98,9 +98,17 @@ type CardMeta = CardRootMeta & {
   supporting?: string[];
 };
 
-type ChefCardMeta = {
-  bio: string;
-};
+interface ChefCardMeta {
+  bio?: string;
+  foregroundHex: string;
```

Comment:
Reading others in this schema, do we want foregroundHex, backgroundHex and paletteId to be optional as well?

Reply:
Also, do we want to add them in CardRootMetaData as well? 
https://github.com/guardian/facia-tool/blob/5cd8690d07d8c61b0985a68ac0de8a6e433dfc56/fronts-client/src/types/Collection.ts#L44

Reply:
Agreed re: palette type – 719c9ded2b9c2548620d290e70fc64af5a5b7413.

Thanks for spotting that @Divs-B – I've adjusted the types slightly and made sure the relevant fields are optional. I've also used a nested image type to avoid the transformation between `image: { src: string }` and `imageSrc: string ...` in the form, which has simplified things a bit.

The `chefImageOverride` property will be able to have a more sensible name once we move to a sum type (as mentioned in the PR description.)

Reply:
I've followed up by nesting the palette field in [53b15b5](https://github.com/guardian/facia-tool/pull/1589/commits/53b15b53bb8d5445706c9a8b35924b9fa1208c21). I think generally I've been operating with the mindset that we need to maintain as close a shape as possible to the original tooling for compatibility reasons, but your comments have made me realise that this is not the case, and ultimately the original model is problematic in part for the reasons you mention above – so thanks for the feedback!